Tests/voltLib/parser_test.py

```diff
@@ -683,6 +683,24 @@ def test_substitution_process_marks(self):
             (lookup.name, process_marks),
             ("SomeSub", "SomeMarks"))
 
+    def test_substitution_mark_glyph_set(self):
+        [group, lookup] = self.parse(
+            'DEF_GROUP "SomeMarks" ENUM GLYPH "acutecmb" GLYPH "gravecmb" '
+            'END_ENUM END_GROUP\n'
+            'DEF_LOOKUP "SomeSub" PROCESS_BASE '
+            'PROCESS_MARKS MARK_GLYPH_SET "SomeMarks" \n'
+            'DIRECTION RTL\n'
+            'AS_SUBSTITUTION\n'
+            'SUB GLYPH "A"\n'
+            'WITH GLYPH "A.c2sc"\n'
+            'END_SUB\n'
+            'END_SUBSTITUTION'
+        ).statements
+        mark_glyph_set = lookup.mark_glyph_set
+        self.assertEqual(
+            (lookup.name, mark_glyph_set),
+            ("SomeSub", "SomeMarks"))
```

Comment:
`mark_glyph_set = lookup.mark_glyph_set` is not needed, use `lookup.mark_glyph_set` directly in the assertEqual.

Reply:
I just copied the test before it, but it makes sense.